airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -147,17 +154,23 @@ class IncrementalMessageStream(SlackStream, ABC):
 
     def __init__(self, default_start_date: DateTime, **kwargs):
         self._start_ts = default_start_date.timestamp()
+        self.set_sub_primary_key()
         super().__init__(**kwargs)
 
+    def set_sub_primary_key(self):
+        if isinstance(self.primary_key, list):
```

Comment:
Please handle a case when self.primary_key is not the 'list' type. Maybe add some exception

Reply:
Added error logger message